apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/ExperimentsResource.java

```diff
@@ -1,6 +1,7 @@
 package com.comet.opik.api.resources.v1.priv;
 
 import com.codahale.metrics.annotation.Timed;
+import com.comet.opik.api.DatasetIdentifier;
```

Comment:
Good catch, thank you, I updated Open API schema. 
I can't delete/replace old `DatasetIdentifier` since it's field has different name: `datasetName` vs `name`. And it's a part of contract:(